lib/vitest.setup.ts

```diff
@@ -26,12 +28,10 @@ Amplify.configure({
   Auth: AUTH_CONFIG,
 });
 
-vi.mock("kafkajs", async (importOriginal) => {
-  return {
-    ...(await importOriginal<typeof import("kafkajs")>()),
-    Kafka: mockedKafka,
-  };
-});
+vi.mock("kafkajs", async (importOriginal) => ({
+  ...(await importOriginal<typeof import("kafkajs")>()),
```

Comment:
Our PR's will step on toes a little bit here but if you get yours in first I'll just change mine since my PR is smaller